docs/src/appendix/sncast/declare-deploy.md

```diff
@@ -0,0 +1,50 @@
+# `declare-deploy`
+Send subsequent declare and deploy transactions of Cairo contract to Starknet at once.
```

Comment:
`at once` kind of suggest we do some kind of batching. But I assume these are still two separate transactions.